pkg/distro/distro.go

```diff
@@ -120,7 +122,9 @@ type ImageType interface {
 	// specified in the given blueprint; it also returns any warnings (e.g.
 	// deprecation notices) generated by the manifest.
 	// The packageSpecSets must be labelled in the same way as the originating PackageSets.
-	Manifest(bp *blueprint.Blueprint, options ImageOptions, repos []rpmmd.RepoConfig, seed int64) (*manifest.Manifest, []string, error)
+	// A custom seed for the rng can be specified, if nil the seed will
+	// be random.
```

Comment:
Maybe we can be more accurate and informative here and say that it will be derived from the current time.

Reply:
Thanks! I'm happy to do that, I was thinking of using crypto.Rand to seed the generator but maybe I'm overthinking it. Happy to either document or change.

Reply:
Not important.  Can do later or not at all.